src/_posts/security/procedures/2000-01-01-gdpr.md

```diff
@@ -0,0 +1,21 @@
+---
+title: GDPR Procedures
+nav: GDPR
+modified_at: 2022-03-15 00:00:00
+tags: procedures gdpr
+---
+
+## How to sign a GDPR DPA with Scalingo?
+
+We have made it easy for you to comply with your own obligations. If you are a
+Scalingo customer and have determined that you qualify as a Data Controller
+under the GDPR, you may need a Data Processing Agreement (DPA) in place with
+Scalingo as a qualifying vendor. We’ve made that part of the process easy for
+you.
+
+This is all you need to do:
+
+- Go [here](https://cdn.scalingo.com/pdf/scalingo-dpa-customer-20180523.pdf) to find our GDPR-compliant DPA, which has been pre-signed on behalf of Scalingo.
+- To complete the DPA, you should fill in the information in the "On behalf of the Customer" box and sign on Page 8.
+- Submit the completed and signed DPA to Scalingo to [dpo@scalingo.com](mailto:dpo@scalingo.com).
+- Upon submitting the validly completed DPA to the email address provided above, the DPA will become legally binding.
```

Comment:
`validly` does not look like a correct English word. I'm not sure what you mean. Is it `valid`?

Reply:
I didn't proofread the text, it is extracted from here https://scalingo.com/blog/gdpr

Google Translate gives me the following translation which seems to the point 
> Après avoir soumis le DPA dûment rempli à l'adresse e-mail fournie ci-dessus, le DPA deviendra juridiquement contraignant.